source/index.txt

```diff
@@ -1,80 +1,247 @@
+            <mongosh-role-management-methods>`.
+
+            Create and update users, authenticate users, and manage user
+            roles with :ref:`User Management Methods 
+            <mongosh-user-management-methods>`. 
+
+      .. image:: /images/mongosh-plp/Technical_SOFTWARE_Terminal(3)_Spot.png
+         :alt: Abstract illustration showing a terminal performing operations
 
-.. default-domain:: mongodb
+   .. tab:: Write Scripts
+      :tabid: scripts
 
-.. contents:: On this page
-   :local:
-   :backlinks: none
-   :depth: 1
-   :class: singlecol
+      .. procedure::
 
-The |mdb-shell|, ``mongosh``, is a fully functional JavaScript and 
-Node.js 16.x |repl| environment for interacting with MongoDB 
-deployments. You can use the |mdb-shell| to test queries and operations 
-directly with your database.
+         .. step:: Perform Common Functions with Scripts
 
-:binary:`~bin.mongosh` is available as a standalone package in the
-`MongoDB Download Center
-<https://www.mongodb.com/try/download/shell?jmp=docs>`__.
+            Write scripts for the |mdb-shell| that modify data in MongoDB or
+            perform administrative operations.
 
-Download and Install ``mongosh``
---------------------------------
+            Explore a tutorial that uses the |mdb-shell| with JavaScript to
+            access MongoDB: :ref:`mdb-shell-write-scripts`.
 
-To learn how to download and install the ``mongosh`` binary, see
-:ref:`mdb-shell-install`.
+         .. step:: Package and Distribute Snippets
 
-Connect to a MongoDB Deployment
--------------------------------
+            You may want to package and distribute your scripts as snippets
+            for you and others to use in their projects. 
 
-Once you have installed the |mdb-shell| and added it to your system
-``PATH``, you can connect to a MongoDB deployment. To learn more, see
-:ref:`mdb-shell-connect`.
+            Learn more about requiring external files and modules in your 
+            scripts: :ref:`mongosh-require-external-files-and-modules`. 
 
-The |mdb-shell| versus the Legacy ``mongo`` Shell
--------------------------------------------------
+         .. step:: Code Scoping and Other Considerations
 
-The legacy ``mongo`` shell was deprecated in MongoDB 5.0 and removed in
-MongoDB 6.0. The new MongoDB Shell, ``mongosh``, offers numerous
-advantages over the legacy shell. The new shell has improved:
+            Learn more about code scope when your scripts load JavaScript 
+            into :binary:`~bin.mongosh`: :ref:`mongosh-scoping`.
 
-- compatibility with the MongoDB `Node.js
-  <https://www.mongodb.com/docs/drivers/node/current/>`__ driver
-- syntax highlighting
-- command history
-- logging
+            Or learn how to work with database query results in class 
+            constructor functions, non-async generator functions, or callbacks
+            to ``.sort()`` on an array: :ref:`mongosh-script-considerations`.
```

Comment:
I’m not sure if this is really something we want to talk about in a landing page? This is intended for users who may have existing scripts for the legacy shell that don’t work anymore in mongosh and doesn’t really target new users (which I assume the landing page here does).

Reply:
+1 to leaving this out!